daisy/workflow/workflow.go

```diff
@@ -245,6 +257,49 @@ func (w *Workflow) populateStep(step *Step) error {
 	return step.SubWorkflow.workflow.populate()
 }
 
+func (w *Workflow) populateMergeWorkflow(step *Step) error {
```

Comment:
Not as much as you might think. It does things just a little bit different. I tried pulling some out into functions but it actually made things more difficult because of the difference in w and step.workflow